src/main/scala/viper/gobra/frontend/Config.scala

```diff
@@ -302,7 +321,19 @@ class ScallopGobraConfig(arguments: Seq[String], isInputOptional: Boolean = fals
     validateFilesExist(include)
     validateFilesIsDirectory(include)
   }
-  validateInput(input, module, include)
+
+  val cutInputWithIdxs: ScallopOption[List[(String, List[Int])]] = input.map(_.map{ arg =>
```

Comment:
`idx` means line numbers here, right? This could be made more clear with a short description